webrender/src/renderer.rs

```diff
@@ -841,10 +866,25 @@ impl Renderer {
             for update in update_list.updates {
                 match update.op {
                     TextureUpdateOp::Create(width, height, format, filter, mode, maybe_bytes) => {
+                        // Create a new native texture, as requested by the texture cache.
+                        let texture_id = self.device
+                                             .create_texture_ids(1, TextureTarget::Default)[0];
+
+                        let CacheTextureId(cache_texture_index) = update.id;
+                        if self.cache_texture_id_map.len() == cache_texture_index {
+                            // It was a new texture, so add to end of the map.
+                            self.cache_texture_id_map.push(Some(texture_id));
+                        } else {
+                            // It was re-using an item from the free-list, so store
+                            // the new ID there.
+                            debug_assert!(self.cache_texture_id_map[cache_texture_index].is_none());
+                            self.cache_texture_id_map[cache_texture_index] = Some(texture_id);
+                        }
```

Comment:
It looks like grow_set was deprecated (https://github.com/rust-lang/rust/pull/17029) - it doesn't seem to be available in stable rust.